config/suprimecam/isr.py

```diff
@@ -11,24 +11,24 @@
     # Crosstalk coefficients for SuprimeCam, as crudely measured by RHL
     config.isr.crosstalk.coeffs = [
         0.00e+00, -8.93e-05, -1.11e-04, -1.18e-04,
-        -8.09e-05,  0.00e+00, -7.15e-06, -1.12e-04,
-        -9.90e-05, -2.28e-05,  0.00e+00, -9.64e-05,
-        -9.59e-05, -9.85e-05, -8.77e-05,  0.00e+00,
-        ]
+        -8.09e-05, 0.00e+00, -7.15e-06, -1.12e-04,
+        -9.90e-05, -2.28e-05, 0.00e+00, -9.64e-05,
+        -9.59e-05, -9.85e-05, -8.77e-05, 0.00e+00,
+    ]
 
 # Crosstalk coefficients derived from Yagi+ 2012
 config.isr.crosstalk.coeffs.crossTalkCoeffs1 = [
-    0,   -0.000148, -0.000162, -0.000167,   # cAA,cAB,cAC,cAD
+    0, -0.000148, -0.000162, -0.000167,   # cAA,cAB,cAC,cAD
     -0.000148, 0, -0.000077, -0.000162,     # cBA,cBB,cBC,cBD
     -0.000162, -0.000077, 0, -0.000148,     # cCA,cCB,cCC,cCD
-    -0.000167, -0.000162, -0.000148, 0,     # cDA,cDB,cDC,cDD 
-    ]
+    -0.000167, -0.000162, -0.000148, 0,     # cDA,cDB,cDC,cDD
+]
 config.isr.crosstalk.coeffs.crossTalkCoeffs2 = [
-    0,       0.000051,0.000050,0.000053, 
-    0.000051,0,       0,       0.000050, 
-    0.000050,0,       0,       0.000051, 
-    0.000053,0.000050,0.000051,0,
-    ]
+    0, 0.000051, 0.000050, 0.000053,
+    0.000051, 0, 0, 0.000050,
+    0.000050, 0, 0, 0.000051,
+    0.000053, 0.000050, 0.000051, 0,
+]
```

Comment:
Yeah, I'm never quite sure about those. It is a danger of running autopep8 on something. I'm apt to leave it and let someone reformat it later if they want to.